src/main/java/uk/gov/register/presentation/config/Field.java

```diff
@@ -32,7 +34,7 @@ public Field(@JsonProperty("field") String fieldName,
     }
 
     public Cardinality getCardinality() {
-        return cardinality;
+        return Stream.of(Cardinality.values()).filter(c -> Objects.equals(c.getId(), cardinality)).findFirst().get();
```

Comment:
wouldn't it be better to do this in the constructor rather than constructing a potentially-invalid object and blowing up only when someone calls getCardinality?